### DISCUSSION TITLE
Flsun v400 profile discussion

### DISCUSSION THREAD

**Discussion by Jokiblack** [OrcaSlicer/OrcaSlicer#772] [General]
With the latest beta version of Orca, I ran some tests with my Flsun V400. Here's a profile that produced an almost perfect dice at a considerable speed. The default speed is 30ms, while this one runs at almost 400.
[0.15mm Standard @FLSun - Copiar.zip](https://github.com/SoftFever/OrcaSlicer/files/11244909/0.15mm.Standard.%40FLSun.-.Copiar.zip)

**Comment by Jokiblack** [comment]
[FLSun V400 0.4 nozzle - Copiar.zip](https://github.com/SoftFever/OrcaSlicer/files/11244917/FLSun.V400.0.4.nozzle.-.Copiar.zip)

**Reply by enewbold** [reply]
Where do we put these .json files? Thank you.

**Reply by enewbold** [reply]
Just upgraded to v 1.63 and lost all of my settings, images, etc.  Setup locking at select filament and won't advance past that. What a frigging mess!

**Reply by enewbold** [reply]
Finally deleted everything Orca and started over.  Reinstalled everything, shutdown and restarted the PC.  Most things back to normal.  Now, can someone advise me where to place these Copiar files so I don't goof everything up again?  Thanks very much.

**Reply by enewbold** [reply]
After upgrading to Orca v 1.6.3, everything is finally working really good now. Orca slicer is the best!

**Comment by SoftFever** [comment]
Nice!
I'm preparing the beta release now.
I will convert this to a discussion to collect feedback and bug reports

**Reply by TonyK54** [reply]
@SoftFever , I have an update I'd like to get in. It adds the Creality Ender 5 Pro (2019 model) as a selectable printer. Please advise if I need to create a new branch for review and a  pull request. Thank you,

**Comment by Jokiblack** [comment]
![16817408084376423586069998390420](https://user-images.githubusercontent.com/8556969/232511509-da23e123-cc34-42ff-8f41-1462aa929ba2.jpg)
dIt doesn't matter what fan configuration you put, small areas always go wrong, in cure this problem does not happen with the fan at 100%

**Reply by SoftFever** [reply]
Cooling is not enough. What cooling fan does V400 use?
You can also try to increase the min layer time in the filament->cooling tab
or slow down for small perimeters
![image](https://user-images.githubusercontent.com/103989404/232937176-678d208b-f6b5-4e64-bf6c-15398b21c07c.png)

**Comment by MrGnarly90** [comment]
i am up and running 
![image](https://user-images.githubusercontent.com/128745709/232540919-11709d8d-25d2-441b-82d4-ff34417dfb6a.png)
p1p speeds modified SR

**Comment by wagzledz** [comment]
I will convert my prusa settings manually and report back with my findings.   I have multiple profiles made for the v400.

**Comment by MrGnarly90** [comment]
![image](https://user-images.githubusercontent.com/128745709/232575326-8d0e2bbe-b229-4e42-9d2b-68f85d09a7fc.png)
SR profile complete, works well

**Reply by xCiNx** [reply]
can you share your profile for a complete noob like me ? :D thank you very much :)

**Reply by Flashhhka** [reply]
Can you share your config files?

**Reply by Hatk1** [reply]
Can you share your config files?

**Reply by docemiel** [reply]
Good afternoon! I come this way to ask the same question! It would be possible to share this profile and configurations in order to have a base for mine, not to mention that table base from flsun which is much prettier. I know that later I have to make some changes to mine... but that would be something... if it were possible, I would appreciate it. Thanks

**Comment by wagzledz** [comment]
where exactly did you folks add the suggested klipper add-in's in your CFG? 

anywhere above the Green "DO NOT TOUCH" settings? lol

**Comment by reify3d** [comment]
So far it is working well on my V400. It even reproduced the Arachne bug from PrusaSlicer. https://github.com/prusa3d/PrusaSlicer/issues/10309

**Comment by wagzledz** [comment]
as seen here, I’ve adjusted the originally provided profile a good bit.   This is a custom 40x40 cube with 5% gyroid infill done with 3 walls, 3 top and bottom layers and the 1 wall first layer and top layer option. 

I’ve adjusted my accel in the speed tab to match my speeder pads settings.  Including accel to decel ratio.  Turned off “slow down for better cooling” in the filament section under the cooling tab and set cooling fans to run at 100% power after first layer.  

Right now I can beat my prusa times by a few minutes.   Definitely fine tuning this beast to make nice prints in pretty good timing.   

Thanks @SoftFever for bringing (what I feel is the best slicer available) to those of us with a v400.  

![18C2A669-E3BE-41A2-A016-30366D14A097](https://user-images.githubusercontent.com/101683623/232940731-8f8b2577-ff36-46cf-8073-99db8802d80d.jpeg)
![4AC1B2BF-A383-4659-AF1F-43E256DF3BC6](https://user-images.githubusercontent.com/101683623/232940734-f628a726-7b26-4c65-9794-bb82cb840424.jpeg)
![61F33DDB-EDDE-42B4-96A7-BDBD92CCE0E2](https://user-images.githubusercontent.com/101683623/232940737-fb77187f-6b02-4edc-bbad-f44a2a7f5526.jpeg)
![E8514A56-01CB-4D0D-BEB6-19A7C0A25299](https://user-images.githubusercontent.com/101683623/232940740-adf8e86c-68f9-4b92-bcf5-9db36c232e22.jpeg)

**Comment by Guilouz** [comment]
This is my profiles (0.12mm, 0.16mm and 0.20mm) if someone want to try :

[Profils V400 OrcaSlicer.zip](https://github.com/SoftFever/OrcaSlicer/files/11301702/Profils.V400.OrcaSlicer.zip)


<img width="2560" alt="Capture d’écran 2023-04-22 à 17 52 28" src="https://user-images.githubusercontent.com/12702322/233794291-d31640c3-da9d-4c68-af2c-d56c5f31128e.png">

**Reply by docemiel** [reply]
Hello Guilouz! And for the FLSun SR? Or you have in your page to the flsun SR!
Thanks for your job and help

**Reply by Guilouz** [reply]
For SR you can just add V400 profiles and modify bed size and adjust retraction settings.

**Reply by Father-Mars** [reply]
What speed/accel settings should i use? the numbers are all over the place it seems like

**Reply by Naninani** [reply]
@Guilouz do you have a PETG profile that I could test? I ran the ones you attached for PLA and they are a great starting point.
I mainly test with cookie cutters that are with 2 walls and 1 wall for cutting edge .44 layer width and that requires fine tuning to have better layer adhesion.
The one I tested fell apart, could be material or not enough cooling, as I forgot to set the part cooling fan to 100%.
Otherwise profiles are quite good.

**Reply by mmunchinski** [reply]
I notice you are calling for higher jerk in your Speed section than your machine limits allow.  Is this intentional?  Those all end up getting capped at the machine limits.

**Reply by nialabert** [reply]
Thanks,
I was able to load the profile for the machine, the filament but not the process. Any Idea ?!?

On the machine settings, Start gcode. You are using a standard Gcode and not one using the klipper macros. Why?
You describe it using the klipper macro on your github "[Klipper-Flsun-Speeder-Pad](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad)"

I'm also in trouble of how is the correct way to activate some options that are in klipper option and in slicer option. Like presssure advance or retractation.

**Reply by Guilouz** [reply]
Use my new profile files for 1.6.3 here : https://drive.google.com/file/d/1xtUaeb0eXWy_ata6oAQTRfSkebOTCn5j/view?usp=share_link

**Reply by junglecat74** [reply]
This profile is a very good starting point.
By any chance: Do you have recommendations / a profile for PETG as well?

Thank you and keep up the AWESOME work :).

**Reply by Father-Mars** [reply]
Am i missing something here? Same model. Same speed and strength settings. 4 hours in Cura. 8.5 hours in orca. What am i doing wrong? 

(The orca screenshot of orca says 15 hours but that was before i changed infill from gyroid to grid. Even still, Cura only had it at 8 hours when using gyroid. ![image](https://github.com/SoftFever/OrcaSlicer/assets/111043488/e795e632-b24b-421e-bf0f-f36f380e92f1)   ![image](https://github.com/SoftFever/OrcaSlicer/assets/111043488/747553b7-4301-48cf-b1f8-3ea0b84054ae) ![image](https://github.com/SoftFever/OrcaSlicer/assets/111043488/8dc7eb04-d0f4-4868-9ef7-02a75b84af3f)

**Reply by Piraye2** [reply]
salut à tous. Je débute avec la flsun v400. Hier soir j'ai mis à jour le speederpad selon les préconisations de wanhao france pour passer la machine à la version V600. Jusqu'à maintenant j'utilisais cura sur ma précédente imprimante mais je n'entends que du bien d'orcaslicer du coup j'ai envie de l'essayer.
Je voudrais optimiser les paramètres de la machine et le slicer. J'ai orca 1.9. Le fichier de configuration existe t'il? Je n'ai trouvé que pour la version 1.6 (merci guilouz).
Quelqu'un pourrait il m'aider pas à pas pour faire les choses correctement et sans tout casser? 
Merci beaucoup
Accessoirement si vous venez en vacances à l'ile de la réunion vous pouvez passer à la maison pour me faire un cours en live )))))

**Comment by SoftFever** [comment]
Thank you everyone for sharing your feedback and configs.
I'm going to update the default profile based on your feedback.
Let me know what do you think of these default settings.
<img width="511" alt="Screenshot 2023-04-23 at 11 19 36 AM" src="https://user-images.githubusercontent.com/103989404/233817987-60a83eb2-3d7d-46cb-83d4-c90ad41548d9.png">

<img width="403" alt="Screenshot 2023-04-23 at 11 18 18 AM" src="https://user-images.githubusercontent.com/103989404/233817996-ba6d785e-7b35-4171-beb0-10ea1a0748b6.png">
<img width="406" alt="Screenshot 2023-04-23 at 11 19 21 AM" src="https://user-images.githubusercontent.com/103989404/233818001-f6d49459-3bb7-4b74-aa39-57951058f9c9.png">

**Reply by Father-Mars** [reply]
This looks awesome, it'd be greatly appreciated if you added one of these for the Super Racer

**Reply by SoftFever** [reply]
I can make an attempt.

**Reply by Father-Mars** [reply]
This is greatly appreciated. I'll talk to a guy who does dev for Klipperized SRs and see if he has any input if that helps!

**Reply by SoftFever** [reply]
I saw @MrGnarly90 has already made one.  #778
I will wait for his response first.
It would be better if the profile is from an actual printer user.

**Reply by Naninani** [reply]
> Thank you everyone for sharing your feedback and configs. I'm going to update the default profile based on your feedback. Let me know what do you think of these default settings. <img alt="Screenshot 2023-04-23 at 11 19 36 AM" width="511" src="https://user-images.githubusercontent.com/103989404/233817987-60a83eb2-3d7d-46cb-83d4-c90ad41548d9.png">
> 
> <img alt="Screenshot 2023-04-23 at 11 18 18 AM" width="403" src="https://user-images.githubusercontent.com/103989404/233817996-ba6d785e-7b35-4171-beb0-10ea1a0748b6.png">
> 
> <img alt="Screenshot 2023-04-23 at 11 19 21 AM" width="406" src="https://user-images.githubusercontent.com/103989404/233818001-f6d49459-3bb7-4b74-aa39-57951058f9c9.png">

@SoftFever I’d suggest taking the profiles from @Guilouz https://github.com/SoftFever/OrcaSlicer/discussions/772#discussioncomment-5683751. He is the flsun guru. The setting that you shared above does not seem correct. 

You may check his flsun repo as well: https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad

**Reply by SoftFever** [reply]
@Naninani, can you point out which value is not correct for you? I can make a change. As as default profile, we in general avoid aggressive settings.

I noticed @Guilouz did a great job creating good Flsun profiles. I want to avoid copying his profile cause it will steal his credit.
He or anyone in the community is welcome to submit updates/changes at any time when they feel ready/comfortable.

**Reply by Guilouz** [reply]
@SoftFever I can make a PR if you want.

**Reply by Hatk1** [reply]
Good evening everyone, I have some problems using Orcaslicer with my stock Flsun Super Racer, I've done all the various calibrations, but when it prints the walls it goes jerky (it's not fluid), so the walls are wavy.  I don't have this problem using Cura.  Is there anyone with the same problem as me who solved it?

**Reply by Father-Mars** [reply]
> @SoftFever I can make a PR if you want.

hey, could you make one for the Super Racer? we need it bad lol

**Reply by pranas** [reply]
> @SoftFever I can make a PR if you want.

@Guilouz Did you submit a PR? Is your profile already included in the latest version of OrcaSlicer?

**Comment by enewbold** [comment]
I am getting a lot of nozzle scraping with my Flsun V400 using Orca v1.6.3.  Where would you suggest I look to try to cure this? The machine is set just fine, mechanically, with the x-offset dialed in on the money.  Your thoughts?  Thanks.

**Reply by TonyK54** [reply]
It is configured under extruder on the printer. These are my settings. 

![image](https://github.com/SoftFever/OrcaSlicer/assets/5837306/8027ee44-1442-4af6-9fcf-427014d33209)

**Comment by enewbold** [comment]
The only change from my config is "Z hop when retracting".  Mine was set to 0.4.  I'll change to 0.6 as depicted above. Thank you.

**Comment by FatFingerGuns** [comment]
What would a good TPU setting be?? Can i use the above setting to print with it?
